### PR TITLE
Add checkpoint downloader script.

### LINKED ISSUE #127: How can I download the pre-trained models?
I would like to download all multi-track pre-trained models, but when I clicked one of the links, it displayed the following error message:
```
<Error>
<Code>NoSuchKey</Code>
<Message>The specified key does not exist.</Message>
<Details>
No such object: magentadata/js/checkpoints/music_vae/multitrack_med_chords_fb256
</Details>
</Error>
```

### PR DESCRIPTION
**Pull request by adarob**
Closes #127.

### CODE REVIEW

File: scripts/checkpoint_downloader.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+"""
+Download all necessary files from a Magenta.js checkpoint URL.
+
+Example usage:
+
+$ python3 checkpoint_downloader /checkpoint/url/ /path/to/output
```
Comment (cghawthorne) [inline scripts/checkpoint_downloader.py]: Any reason for the python3 vs python mismatch between this and the script?

File: scripts/checkpoint_converter.py [hunk 2]
```diff
@@ -22,7 +22,7 @@
 
 Example usage:
 
-python3 checkpoint_converter /path/to/checkpoint.ckpt /path/to/output \
+$ python checkpoint_converter /path/to/checkpoint.ckpt /path/to/output \
```
Comment (notwaldorf) [inline scripts/checkpoint_converter.py]: why is this `python` but the instructions in the downloader say `python3 checkpoint_downloader /checkpoint/url/ /path/to/output`? I know 0 python

Comment (adarob) [reply scripts/checkpoint_converter.py]: oversight

File: scripts/checkpoint_downloader.py (hunk 1 above)
Comment (adarob) [reply scripts/checkpoint_downloader.py]: oversight

**Pull request merged**